pkgs/by-name/as/aseprite/package.nix

```diff
@@ -33,15 +33,37 @@
 
 clangStdenv.mkDerivation (finalAttrs: {
   pname = "aseprite";
-  version = "1.3.7";
+  version = "1.3.13";
 
-  src = fetchFromGitHub {
-    owner = "aseprite";
-    repo = "aseprite";
-    rev = "v" + finalAttrs.version;
-    fetchSubmodules = true;
-    hash = "sha256-75kYJXmyags0cW2D5Ksq1uUrFSCAkFOdmn7Ya/6jLXc=";
-  };
+  srcs = [
+    (fetchFromGitHub {
+      name = "aseprite-source";
+      owner = "aseprite";
+      repo = "aseprite";
+      rev = "v${finalAttrs.version}";
+      fetchSubmodules = true;
+      hash = "sha256-eeB/4fQp1lbNYQj9LpNhOn7DYxaTc+BcmyvY2vPzpxk=";
+    })
+
+    # translation strings
+    (fetchFromGitHub {
+      name = "aseprite-strings";
+      owner = "aseprite";
+      repo = "strings";
+      rev = "7b0af61dec1d98242d7eb2e9cab835d442d21235";
+      hash = "sha256-8OwwHCFP55pwLjk5O+a36hDZf9uX3P7cNliJM5SZdAg=";
+    })
+  ];
+
+  sourceRoot = "aseprite-source";
+
+  postUnpack = ''
+    # Link additional translations. Note: Although there's an English
+    # translation in the strings repo, it might not be up-to-date with the
+    # existing file, so it's important to not clobber the existing file when
+    # bringing in the translations.
+    cp --symbolic-link --no-clobber $PWD/aseprite-strings/* ./aseprite-source/data/strings
```

Comment:
Removing `--symbolic-link` here should fix translations.  May also want to update the comment since it's being copied rather than linked